benefits/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,60 @@
+<link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap" rel="stylesheet">
+<style>
+  body {
+    color: #535456;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 14px;
+  }
+
+  h1 {
+    color: #212121;
+    font-size: 24px;
+  }
+
+  a {
+    color: unset;
+  }
+
+  .link-wrapper {
+    padding: 16px 0px;
+  }
+
+  .link-button {
+    background: #025B86;
+    border-radius: 16px;
+    color: #fff;
+    font-weight: 700;
+    padding: 8px 16px;
+    text-decoration: none;
+  }
+
+  .divider {
+    border: 2px solid #ECEFF1;
+  }
+</style>
+
+{% autoescape off %}
+  <h1>Your reset password link</h1>
+  <p>
+    Click the link below to reset the password for your Cal-ITP Benefits Administrator account.
```

Comment:
This is a nit to be sure, but I know we've been discussing consistency for this PR...

We say `Your reset password link` and `Click the link below...` but then later say `If the button above doesn't work, paste this link...`

Should we use `button` instead of `link` when we're directing the action towards the button? And reserve usage of `link` for the fallback link?

@cmajel